starfish/experiment/builder/__init__.py

```diff
@@ -137,6 +168,7 @@ def build_image(
                 image.shape,
                 extras=image.extras,
             )
+            tile.filepath = image.filepath
```

Comment:
I'm not sure I understand this? Are you setting tile.filepath every-time? If so wouldn't we always hit the DevNull case?

Reply:
The `filepath` property in the modified [starfish/experiment/builder/providers.py ](https://github.com/spacetx/starfish/pull/1053/commits/511a5b9dc5ac7b717326b350baa154a8f977f63b#diff-36f200ab95f1d7cb24a4d35429e88ee1R53) defaults to `None`. So the DevNull case will only happen if the `TileFetcher` implementation explicitly sets a value to `filepath` for a  `FetchedTile`.

Reply:
ahh ok that makes sense, thanks!